app/src/ui/branches/branches-container.tsx

```diff
@@ -161,11 +161,13 @@ export class BranchesContainer extends React.Component<
 
     return (
       <Row className="merge-button-row">
-        <Button className="merge-button" onClick={this.onMergeClick}>
+        <Button
+          className="merge-button"
+          onClick={this.onMergeClick}
+          tooltip={`Choose a branch to merge into ${currentBranch.name}`}
+        >
           <Octicon className="icon" symbol={OcticonSymbol.gitMerge} />
-          <span title={`Merge a branch into ${currentBranch.name}`}>
-            Choose a branch to merge into <strong>{currentBranch.name}</strong>
-          </span>
+          Choose a branch to merge into <strong>{currentBranch.name}</strong>
```

Comment:
Should be one of our regular tooltips anyways.. I just updated the text to match the button.

Before:
![image](https://github.com/desktop/desktop/assets/75402236/fc9c62cf-a155-47bb-9369-d0f5dada7b5c)

After:
![image](https://github.com/desktop/desktop/assets/75402236/76f15000-3cd4-4188-a20f-aab1d9c3ff0f)

Reply:
💯